src/main/java/uk/gov/justice/laa/crime/cfecrime/utils/RequestHandlerUtil.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.justice.laa.crime.cfecrime.utils;
+
+import lombok.experimental.UtilityClass;
+import lombok.extern.slf4j.Slf4j;
+import uk.gov.justice.laa.crime.cfecrime.api.Result.Outcome;
+import uk.gov.justice.laa.crime.cfecrime.api.*;
+
+@Slf4j
+@UtilityClass
+public class RequestHandlerUtil {
+
+    public static CfeCrimeResponse handleRequest(CfeCrimeRequest request){
+        CfeCrimeResponse response = new CfeCrimeResponse();
+        Boolean under18 = false;
+        Boolean passported = false;
+        Outcome oc = Outcome.INELIGIBLE;
+
+        if (request.getSectionUnder18() != null) {
+            under18 = request.getSectionUnder18().getClientUnder18();
+        }
+        if (request.getSectionPassportedBenefit() != null) {
+            passported = request.getSectionPassportedBenefit().getPassportedBenefit();
+        }
+        oc = getOutcomeFromAgeAndPassportedBenefit(under18,passported);
+
+        if (request.getAssessment() != null) {
+            Assessment assessment = request.getAssessment();
+            if (assessment.getAssessmentDate() != null) {
+                oc = Outcome.ELIGIBLE;
+            }
+        }
+
+        Result result = new Result();
+        result.setOutcome(oc);
+        response.withResult(result);
+        return response;
+    }
+
+    /*
+     * get the outcome from
+     * @params clientUnder18, clientPassportedBenefit
+     * @return outcome (if outcome is not determined returns null (for more information)
+     *
+     */
```

Comment:
I'm not really a fan of this sort of documentation. It doesn't really add anything while adding 6 lines to the codebase. It is mostly inferable from the code (especially as the code is so straightforward)